src/main/resources/local.properties

```diff
@@ -0,0 +1 @@
+jetty.port=5040
```

Comment:
This file should be inside .gitignore and removed from the pull request. This file is developer-specific. I would instead create a local.properties.sample that can be included in the repo and each developer can have a copy in his machine under local.properties.